lock/src/main/java/org/infinispan/lock/impl/lock/RequestExpirationScheduler.java

```diff
@@ -0,0 +1,123 @@
+package org.infinispan.lock.impl.lock;
+
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.concurrent.ScheduledExecutorService;
+import java.util.concurrent.ScheduledFuture;
+import java.util.concurrent.TimeUnit;
+
+import org.infinispan.commons.logging.LogFactory;
+import org.infinispan.lock.impl.log.Log;
+
+/**
+ * This class holds the logic to schedule/abort requests that need to be completed at a given time.
+ *
+ * @author Katia Aresti, karesti@redhat.com
+ * @since 9.2
+ */
+public class RequestExpirationScheduler {
+
+   private static final Log log = LogFactory.getLog(RequestExpirationScheduler.class, Log.class);
```

Comment:
You can just use `MethodHandles.lookup().lookupClass()` instead of coding the class name.

Reply:
ok